packages/clerk-js/src/ui/primitives/FormErrorText.tsx

```diff
@@ -10,27 +12,40 @@ const { applyVariants } = createVariants(theme => ({
     willChange: 'transform, opacity, height',
     marginTop: theme.sizes.$2,
     animation: `${animations.textInSmall} ${theme.transitionDuration.$fast}`,
+    display: 'flex',
+    gap: theme.sizes.$2,
+    position: 'absolute',
+    top: '0',
   },
   variants: {},
 }));
 
 type FormErrorTextProps = React.PropsWithChildren<StyleVariants<typeof applyVariants>>;
 
-export const FormErrorText = (props: FormErrorTextProps) => {
+export const FormErrorText = forwardRef<HTMLElement, FormErrorTextProps>((props, ref) => {
   const { hasError, errorMessageId } = useFormControl() || {};
 
   if (!hasError && !props.children) {
     return null;
   }
 
+  const { children, ...rest } = props;
+
   return (
     <Text
+      ref={ref}
       variant='smallRegular'
       colorScheme='danger'
       aria-live='polite'
       id={errorMessageId}
-      {...props}
+      {...rest}
       css={applyVariants(props)}
-    />
+    >
```

Comment:
❓ These icons affect all error messages across ClerkJS screens. Is this intentional? If yes, we need to make sure that errors are rendered correctly.

Reply:
Yes we want to display this icon in all error messages. In the videos above some different errors can be seen as well.

Reply:
This is now optimised